core/static/sass/_all.scss

```diff
@@ -461,7 +461,6 @@ footer {
 }
 #id_conditions_chosen .chosen-choices input[type="text"] {
     height: 26px;
-    width: 15px !important;
```

Comment:
finfine! :grinning:  :+1:

Reply:
la sola problemo kun tio (sed rilatas al `chosen` ne al nia kodo) estas ke la elemento ne plu adaptas sin al la larĝeco de la ekrano. ekzemple, se vi uzas poŝtelefonon en horizontala pozicio kaj turnas ĝin al vertikala, ĉiuj elementoj mallarĝiĝos por adaptiĝi al nova ekrano, krom tiuj modifitaj de `chosen`, kiuj nun etendiĝos trans la limoj de la ekrano. aspektas malbele.

Reply:
Ĉu [`select2`](https://select2.org/) estus pli bona elekto? _(intenca serĉo)_